index.js

```diff
@@ -77,24 +158,29 @@ const logPageLoadTime = async (n, url, now) => {
 
 try {
   if (isGhosteryEnabled) {
-    const addon = await downloadAddon(
-      'https://github.com/ghostery/ghostery-extension/releases/download/v8.9.15/ghostery-firefox-v8.9.15.zip',
-    );
+    if (!fs.existsSync(`profiles/withGhostery/${selectedBrowser}/onboarded`)) {
+      let extension = 'moz-extension';
```

Comment:
Lets move all browser related params to a config object. 

eg. 
```
const config = isFirefox ? {
  extensionScheme: 'moz-extension',
  addonId: 'xx',
} : {
  extensionScheme: 'chrome-extension',
  addonId: 'yy',
};
```

Reply:
Done.